src/types/Constants.ts

```diff
@@ -110,6 +110,7 @@ export enum PAGES {
   COMPANY_SUBSCRIPTIONS = 'company-subscriptions',
   COMPANY_SUBSCRIPTIONS_DETAIL = 'company-subscriptions-detail',
   COMPANY_DATA = 'company_data',
+  ONBOARDING_SERVICE_PROVIDER = 'onboardingServiceProvider',
```

Comment:
Above in Line 105 we already have 

`ONBOARDING_SERVICEPROVIDER = 'onboarding-serviceprovider',`

Is this a duplicate?